site/viamguides/src/air-quality/air-quality.md

```diff
@@ -0,0 +1,340 @@
+    }
+    ```
+    ![configure smart plug](assets/configurePlug.png)
+1. **Save** your updates.
+
+<!-- ------------------------ -->
+## Test the hardware
+Duration: 5
+
+### Test the air sensor
+
+1. In [the Viam app](https://app.viam.com/robots) under the **CONTROL** tab, you can see two hardware components called `generic-1` and `sensor-1`. They are your smart plug and air sensor that you configured on the **CONFIGURE** tab.
+  ![control tab](assets/controlTab.png)
+1. Find the `sensor-1` card, and expand the TEST accordion. The module we added has a `GetReadings` function to get the readings from our air sensor. Select `Manual refresh`, and then click the refresh icon to manually get the readings.
+  ![get readings](assets/getReadings.png)
+    > aside negative
+    > If any problems occur, check under the **LOGS** tab to see what might be going wrong.
+    
+    > aside positive
+    > If you're curious about what these values mean, refer to the U.S. Environmental Protection Agency's [basics on particulate matter (PM)](https://www.epa.gov/pm-pollution/particulate-matter-pm-basics). For measuring the [indoor air quality](https://www.epa.gov/indoor-air-quality-iaq/indoor-pollutants-and-sources) in your home, we'll be looking at PM2.5, a very small particle size.
+
+### Test the smart plug
+
+1. Next, test that your smart plug is working properly. Make sure the fan is still plugged in to the powered smart plug. Also make sure the fan knob is set to on. 
+1. In [the Viam app](https://app.viam.com/robots) under the **CONTROL** tab, find the `generic-1` card, and expand the DOCOMMAND accordion. The module we added has a few functions to control the smart plug. Under the **Input** section, add the following JSON object (if your fan is still running), and then click **Execute** to toggle off the smart plug.
+    ```json
+    {
+      "toggle_off": []
+    }
+    ```
+    ![do command](assets/doCommand.png)
+1. Now update `"toggle_off"` to `"toggle_on"` in the JSON object, and **Execute** once again.
+1. From [the module listing in the registry](https://app.viam.com/module/joyce/kasasmartplug), you may be able to refer to additional documentation in [the module's README](https://github.com/loopDelicious/viam-kasa-switch) to learn about other functions you can use to test your hardware.
+
+<!-- ------------------------ -->
+## Program your air monitoring device
+Duration: 6
+
+At this point, you have configured and tested your machine and peripherals, but nothing is happening automatically. In the next section, program your air monitoring device to be a little smarter. Make your machine periodically get readings, turn on the air filter if the values are high, and turn off again when the values are low.
+
+### Create an automation script
+
+1. To configure the machine to automatically run a command to execute a script, use a [Viam process](https://docs.viam.com/configure/processes/). Create a new file called `process.py`.
```

Comment:
We should probably be specific about where this new file should be created since the reader might still be ssh'd into their Pi:

```suggestion
1. To configure the machine to automatically run a command to execute a script, use a [Viam process](https://docs.viam.com/configure/processes/). Create a new file on your computer called `process.py`.
```